github/License.py

```diff
@@ -160,3 +161,11 @@ def _useAttributes(self, attributes):
             self._conditions = self._makeListOfStringsAttribute(attributes["conditions"])
         if "limitations" in attributes:  # pragma no branch
             self._limitations = self._makeListOfStringsAttribute(attributes["limitations"])
+
+
+class RepositoryLicense(License):
```

Comment:
Part of me questions why you're adding this? If License perfectly encapsulates the data returned, I see no reason why to add it.

Reply:
I guess I have misunderstood how the code works. I will do the requested change and remove RepositoryLicense class, as it does look a bit odd. I understood originally that there is an extra request which is done towards the License API itself without this change, but I admit that I have forgotten to verify this activity.

Reply:
Let me know if there is something else which needs to be fixed.